gateway/Gopkg.toml

```diff
@@ -27,3 +27,7 @@ ignored = ["github.com/openfaas/faas/gateway/queue"]
 [[constraint]]
   name = "github.com/openfaas/faas-provider"
   version = "0.9.1"
```

Comment:
Would you be able to vendor 0.9.2 - the latest version? https://github.com/openfaas/faas-provider/releases/tag/0.9.2